dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/security/annotations/SecurityAnnotationsTest.java

```diff
@@ -0,0 +1,538 @@
+
+import org.apache.wink.client.ClientConfig;
+import org.apache.wink.client.RestClient;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class SecurityAnnotationsTest {
+
+    @Server("com.ibm.ws.jaxrs.fat.security")
+    public static LibertyServer server;
+
+    private static final String secwar = "security";
+    private static final String secNoWebXmlWar = "securityNoWebXml";
+    private static final String clz = "SecurityAnnotationsTest";
+    private static final Logger LOG = Logger.getLogger(SecurityAnnotationsTest.class.getName());
+    private RestClient client;
+
+    private static LibertyServer swapToServer(LibertyServer newServer) throws Exception {
+        LibertyServer oldServer = server;
+        tearDown();
+        server = newServer;
+        return oldServer;
+    }
+
+    @Before
+    public void setUp() {
+        ClientConfig config = new ClientConfig();
+        config.connectTimeout(120000);
+        config.readTimeout(120000);
+        client = new RestClient(config);
+    }
+
+    @BeforeClass
+    public static void setupClass() throws Exception {
+        ShrinkHelper.defaultDropinApp(server, secwar, "com.ibm.ws.jaxrs.fat.security.annotations",
+                                      "com.ibm.ws.jaxrs.fat.security.ssl",
+                                      "com.ibm.ws.jaxrs.fat.securitycontext",
+                                      "com.ibm.ws.jaxrs.fat.securitycontext.xml");
+        ShrinkHelper.defaultApp(server, secNoWebXmlWar, "com.ibm.ws.jaxrs.fat.security.nowebxml.annotations",
+                                "com.ibm.ws.jaxrs.fat.security.nowebxml.ssl",
+                                "com.ibm.ws.jaxrs.fat.securitycontext.nowebxml",
+                                "com.ibm.ws.jaxrs.fat.securitycontext.nowebxml.xml");
+
+        // Make sure we don't fail because we try to start an
+        // already started server
+        try {
+            server.startServer(true);
+            assertNotNull("The server did not start", server.waitForStringInLog("CWWKF0011I"));
+            assertNotNull("The Security Service should be ready", server.waitForStringInLog("CWWKS0008I"));
+        } catch (Exception e) {
+            System.out.println(e.toString());
+        }
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        if (server != null) {
+            server.stopServer();
+        }
+    }
+
+    private static String SECANNO_BASE_TEST_URI = getBaseTestUri(secwar, "nomapper");
+    private static String SECANNO_NOWEBXML_BASE_TEST_URI = getBaseTestUri(secNoWebXmlWar, "nomapper");
+
+    /**
+     * tests DenyAll at class level
+     */
+//    @Test
```

Comment:
What's with all these commented tests?  Are they specified in the Test Servlet?  If so, we can probably delete the comments.  But let's make sure that these tests are actually running first.

Reply:
I found a bunch of tests that are commented out either because behavior changed or maybe the previous team never got them running in the first place. Either way "fixing" them is beyond the scope of this already mammoth task so I left them in. I'll create an issue to investigate/fix/delete them in the future.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/6326